app/views/includes/govuk_wrapper.scala.html

```diff
@@ -52,14 +57,14 @@
 
 @govukLayout(
     pageTitle = Some(pageTitle),
-    headBlock = Some(hmrcHead(head)),
-    headerBlock = Some(hmrcStandardHeader(
-        serviceUrl = None,
-        serviceName = Some(messages("service.name")),
-        displayHmrcBanner = false
-    )),
+    headBlock = Some(hmrcInternalHead(head, nonce = CSPNonce.get)),
+    headerBlock = Some(hmrcInternalHeader(InternalHeader(
+                    homepageUrl = "/",
```

Comment:
this should be relative url to the app or something, now it does nothing but refresh current page

Reply:
see above vid